Tests/BaseTestCase.php

```diff
@@ -32,8 +35,9 @@ public function setUp()
         $config = Setup::createAnnotationMetadataConfiguration($paths, true);
 
         $connectionConfig = $this->getConfig();
-        $this->entityManager = DoctrineEntityManager::initialize($connectionConfig['parameters'], $config);
+        $entityManager = EntityManager::create($connectionConfig['parameters'], $config);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Doctrine\ORM\EntityManager' in method 'setUp'.](https://www.codacy.com/app/Managlea/EntityManager/file/2321555249/issues/source?bid=3149133&fileBranchId=3204686#l38)